### DISCUSSION TITLE
How to track scheduled ground station passes/contact planning?

### DISCUSSION THREAD

**Discussion by YeomansIII** [yamcs/yamcs#583] [Q&A]
I am trying to determine if Yamcs has a way to schedule and track ground station passes/contacts, triggering scripts/commands during a pass. I have been able to schedule an "event" in the future, but an event does not have a start and end time, only a reception/generation time.

Could someone provide details on how they schedule and track their ground station passes with Yamcs?

**Comment by fqqb** [comment]
For any passers-by: this discussion was also opened as an issue: #584
